docs/website-style.md

```diff
@@ -11,83 +11,72 @@ We're still building out custom CSS functionality with the MyST engine.
 
-Normal-styled
-```
+### Use content blocks
 
-Results in:
+[Content blocks](../blocks.md) allow you to attach arbitrary metadata to chunks of content.
+You can attach one or more CSS classes by defining a `class` attribute for a block.
+For example the following:
 
-+++ {"class": "col-gutter-right"}
-Right-styled
+```{myst}
++++ {"class": "text-gradient"}
+This is _emphasized_. This is not emphasized.
 
 +++
 
-Normal-styled
+This is not emphasized.
+```
 
-## Use `div` and `span` elements
+### Use `div` and `span` elements
 
 You can attach classes directly to [`div` and `span` elements](#div-and-span).
 
 {myst:directive}`div` and {myst:role}`span` are analogous to their HTML counterparts. Unlike their directive/role, the HTML elements can also be given `style` options, e.g.
 
-<div class="col-gutter-right" style="font-weight: bold;">Here's my div</div>
-
-Here's some <span class="col-gutter-right" style="font-weight:bold;">Span</span> content
+```{myst}
+<div class="text-gradient" style="font-weight: bold;">Here's my <em>div</em></div>
 
-## Add CSS classes to directives
+Here's some <span class="text-gradient" style="font-weight:bold;">span <em>styled</em></span> content
+```
 
-:::{note} Not all directives support the `:class:` option
-If you wish to attach classes to a directive that doesn't seem to support it, please [open an issue](https://github.com/jupyter-book/mystmd/issues)
-:::
+### Add CSS classes to directives
 
 Many directives and content blocks have a `:class:` option that can be used to add arbitrary CSS classes.
 For example, below we add a CSS class to an admonition directive to snap it to the right:
 
-````md
+````{myst}
 ```{note}
-:class: col-gutter-right
-I'm on the right!
+:class: text-gradient
+I'm _very stylish_.
 ```
 ````
 
-```{note}
-:class: col-gutter-right
-I'm on the right!
-```
-
-## Use the HTML theme grid system classes to position content
-
-The HTML themes come with [a grid system of CSS classes](https://jupyter-book.github.io/myst-theme/?path=/docs/components-grid-system--docs).
-You can use these to position content according to the link above.
-
-## Use Tailwind CSS classes
-
-:::{note} Provide feedback
-This issue tracks loading extra Tailwind CSS classes when they're used on a page:
-
-- https://github.com/jupyter-book/mystmd/issues/1617
-:::
+## Built-in CSS Classes
 
-You can use any [Tailwind CSS class](https://tailwindcss.com/docs/installation) that's loaded on a page to style your content.
-See the Tailwind documentation for examples of how to do this.
-If a class seems to have no effect, it is likely not loaded on the page by MyST.
-Currently, it's not possible to customize which classes are included on a page (see above for an issue tracking this).
+The HTML themes come with [a grid system of CSS classes](https://jupyter-book.github.io/myst-theme/?path=/docs/components-grid-system--docs), which can be used out-of-the-box to position content.
```

Comment:
Maybe share a link to the issue where we discuss adding support for user-given tailwind classes?

Reply:
I'm holding off here because I think we ought to suggest best practice, and what that looks like is evolving? See my comment about e.g. `myst style ./foo.css` etc. :)